packages/dialog/src/vaadin-dialog.js

```diff
@@ -48,16 +48,13 @@ export { DialogOverlay } from './vaadin-dialog-overlay.js';
  *
  * ### Styling
  *
- * `<vaadin-dialog>` uses `<vaadin-dialog-overlay>` internal
```

Comment:
We could probably also remove mention of `theme` propagation to the overlay below, as it was removed for e.g. `vaadin-confirm-dialog`.